DuckDuckGo/NetworkProtectionStatusViewModel.swift

```diff
@@ -57,7 +58,11 @@ final class NetworkProtectionStatusViewModel: ObservableObject {
 
     // MARK: Toggle Item
     @Published public var isNetPEnabled = false
-    @Published public var statusMessage: String
+    @Published public var statusMessage: String {
+        didSet {
+            WidgetCenter.shared.reloadTimelines(ofKind: "VPNStatusWidget")
+        }
+    }
```

Comment:
There is likely a better place for this, this was the quickest approach I hacked together.

Reply:
Yeah, I think there should probably be a new type for this which is observing the status and has a lifecycle tied to the AppDelegate.